src/Job/GenerateVideoThumbnails.php

```
@@ -0,0 +1,189 @@
+        ));
+
+        $this->debugManager->logInfo(sprintf(
+            'Bulk video thumbnail generation completed - Processed: %d, Success: %d, Failed: %d, Skipped: %d',
+            $totalProcessed, $totalSuccess, $totalFailed, $totalSkipped
+        ), DebugManager::COMPONENT_SERVICE, $opId);
+    }
+
+    /**
+     * Process a single media item for video thumbnail generation
+     */
+    protected function processSingleMedia(int $mediaId, bool $forceRegenerate, ?int $percentage, string $opId): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$forceRegenerate'. (Unused Code Rules)

(UnusedFormalParameter)